consensus/vbft/chain_store.go

```diff
@@ -83,7 +89,13 @@ func (self *ChainStore) getExecMerkleRoot(blkNum uint32) (common.Uint256, error)
 	} else {
 		return merkleRoot, nil
 	}
+}
 
+func (self *ChainStore) getCrossStatesRoot(blkNum uint32) common.Uint256 {
+	if blk, present := self.pendingBlocks[blkNum]; blk != nil && present {
+		return blk.execResult.CrossStatesRoot
+	}
+	return common.UINT256_EMPTY
```

Comment:
try getting from ledger if not found in pending blocks

Reply:
fixed